### DISCUSSION TITLE
Text grid image out of alignment.

### DISCUSSION THREAD

**Discussion by neural-oD** [ImageMagick/ImageMagick#3414] [Help] [answered]
Hi All,
I'm running into a slight alignment problem. I'm not enough of a power user of Imagemagick to try and solve this. 
Script I'm running:

`cat test.txt |awk '/Success/,EOF'| tee >(convert -font Lato-Medium -pointsize 14 label:@- img.png)`

this is what it should look like - the output on the terminal:

![2021-03-18_12-49](https://user-images.githubusercontent.com/22800677/111615744-c2c8c780-87e9-11eb-87d3-45093418f68e.png)

However this is what the png image that is produced....

![imagemagick](https://user-images.githubusercontent.com/22800677/111615818-d6742e00-87e9-11eb-8823-e59a057592b7.png)

Any suggestions? - PS - not worried about the background color for now, just the alignment issues.

Thanks for any help

**Comment by snibgo** [comment, marked as answer]
You need to use a monospaced font, where all glyphs take the same width, such as a font from the Courier family.

**Reply by neural-oD** [reply]
Wow - they easy hey.
@snibgo  - you're a legend thank you very much!
